.github/workflows/ci.yml

```
@@ -37,6 +37,13 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install protobuf==3.7.1
+    
+    # Chrome Stable is already installed by default.
+    - name: Install Chrome ToT
+      run: bash lighthouse-core/scripts/download-chrome.sh
+      env:
+        CHROME_PATH: /home/runner/chrome-tot
```

```suggestion
        CHROME_PATH: /home/runner/chrome-linux-tot
```

even if we're not caching, feels like they should match